packages/expo-font/ios/EXFont/EXFontLoader.m

```diff
@@ -1,48 +1,42 @@
 // Copyright 2015-present 650 Industries. All rights reserved.
 
 #import <EXFont/EXFontLoader.h>
-#import <EXCore/EXAppLifecycleService.h>
 #import <EXFont/EXFontLoaderProcessor.h>
 #import <EXFontInterface/EXFontManagerInterface.h>
+#import <EXFont/EXFontScaler.h>
 #import <EXFont/EXFont.h>
 #import <objc/runtime.h>
-#import <EXFont/UIFont+EXFontLoader.h>
 #import <EXFont/EXFontManager.h>
+#import <EXFont/EXFontScalersManager.h>
 
 @interface EXFontLoader ()
 
-@property (nonatomic, assign) BOOL isInForeground;
-@property (nonatomic, weak) id<EXAppLifecycleService> lifecycleManager;
+@property (nonatomic, strong) EXFontScaler *scaler;
+@property (nonatomic, strong) EXFontLoaderProcessor *processor;
 
 @end
 
 @implementation EXFontLoader
 
 EX_EXPORT_MODULE(ExpoFontLoader);
 
-- (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
+- (instancetype)init
 {
-  if (_lifecycleManager) {
-    [_lifecycleManager unregisterAppLifecycleListener:self];
+  if (self = [super init]) {
+    _scaler = [[EXFontScaler alloc] init];
+    _processor = [[EXFontLoaderProcessor alloc] init];
   }
+  return self;
+}
 
-  _lifecycleManager = nil;
-
+- (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
+{
   if (moduleRegistry) {
     id<EXFontManagerInterface> manager = [moduleRegistry getModuleImplementingProtocol:@protocol(EXFontManagerInterface)];
-    [manager addFontProccessor:[[EXFontLoaderProcessor alloc] init]];
-    _lifecycleManager = [moduleRegistry getModuleImplementingProtocol:@protocol(EXAppLifecycleService)];
-  }
+    [manager addFontProccessor:_processor];
```

Comment:
Could you fix the spelling of addFontProccessor (=> addFontProcessor) either in this PR or a separate one?